apps/io-services-cms-webapp/ReviewService/function.json

```
@@ -0,0 +1,21 @@
+{
+    "bindings": [
+        {
+            "authLevel": "anonymous",
+            "type": "httpTrigger",
+            "direction": "in",
+            "name": "req",
+            "route": "services/:serviceId/review",
```

it should be with curly braces:
```suggestion
            "route": "services/{service_id}/review",
```